ads_plugin.py

```diff
@@ -156,7 +156,12 @@ def __init__(self, ortho, *args, **kwargs):
         compute_morphometrics_button = wx.Button(self, label="Compute morphometrics")
         compute_morphometrics_button.SetForegroundColour(button_label_color)
         compute_morphometrics_button.Bind(wx.EVT_BUTTON, self.on_compute_morphometrics_button)
-        compute_morphometrics_button.SetToolTip(wx.ToolTip("Calculates and saves the morphometrics to an excel and csv file.")) 
+        compute_morphometrics_button.SetToolTip(
+            wx.ToolTip(
+                "Calculates and saves the morphometrics to an excel and csv file. "
```

Comment:
Not directly related to this PR but I haven't find how to save the file in csv. The only option I have is xlsx.